test/test-_loadInIframe.js

```diff
@@ -126,6 +126,16 @@ buster.testCase('_loadInIframe',
       assert.equals(this.testObject.window, this._stubs.window);
       assert.equals(this.testObject.document, this._stubs.document);
       assert.equals(this.testObject.$, this._stubs.$);
+    },
+
+    'Throws an exception if $ is not defined on the iframe': function()
+    {
+      assert.exception(function()
+      {
+        this._stubs.$ = undefined;
```

Comment:
Usually, just moving out of assert function is enough. But this case, yes, having extra setUp/tearDown makes sense. Thank you.